techniques/aria/ARIA17.html

```diff
@@ -1,84 +1,110 @@
+      <h2>Examples</h2>
       <section class="example">
          <h3>Social Security Number</h3>
-         
-            <p>Social security number fields which are 9 digits long and broken up into 3 segments can be grouped using role="group".</p>
-            <pre xml:space="preserve">&lt;div role="group" aria-labelledby="ssn1"&gt;
-   &lt;span id="ssn1"&gt;Social Security#&lt;/span&gt; 
-   &lt;span style="color: #D90D0D;"&gt; * &lt;/span&gt;
-   &lt;input size="3" type="text" aria-required="true" title="First 3 digits" /&gt;-
-   &lt;input size="2" type="text" aria-required="true" title="Next 2 digits" /&gt;-
-   &lt;input size="4" type="text" aria-required="true" title="Last 4 digits" /&gt;
+         <p>Social security number fields which are 9 digits long and broken up into 3 segments can be grouped using role="group".</p>
+<pre xml:space="preserve">&lt;div role="group" aria-labelledby="ssn1"&gt;
+  &lt;div id="ssn1"&gt;Social Security Number (required)&lt;/div&gt;
+  &lt;input size="3" type="text" aria-required="true" title="First 3 digits" /&gt; -
+  &lt;input size="2" type="text" aria-required="true" title="Next 2 digits" /&gt; -
+  &lt;input size="4" type="text" aria-required="true" title="Last 4 digits" /&gt;
 &lt;/div&gt;</pre>
-            <p class="working-example">Working example: <a href="../../working-examples/aria-grouping-multipart-fields-ssn/">Multiple part field groups</a>.</p>
-         
+
+         <p class="working-example">Working example: <a href="../../working-examples/aria-grouping-multipart-fields-ssn/">Multiple part field groups</a>.</p>   
       </section>
       <section class="example">
          <h3>Identifying radio groups</h3>
-         
-            <p>This example demonstrates use role=radiogroup. Note also that the radio buttons are custom controls with role=radio. (But the script to make the span actually work like radio buttons is not included in this example. ) One may optionally employ CSS to place a border around groups of such fields to visually reinforce the group relationship. The CSS properties are available below the form.</p>
-            <pre xml:space="preserve">&lt;h3&gt;Set Alerts for your Account&lt;/h3&gt;
-  &lt;div role="radiogroup" aria-labelledby="alert1"&gt;
-    &lt;p id="alert1"&gt;Send an alert when balance exceeds $ 3,000&lt;/p&gt;
-    &lt;div&gt;
-      &lt;span role="radio" aria-labelledby="a1r1" name="a1radio"&gt;&lt;/span&gt;
-      &lt;span id="a1r1"&gt;Yes&lt;/span&gt;
-    &lt;/div&gt;
-    &lt;div&gt;
-      &lt;span role="radio" aria-labelledby="a1r2" name="a1radio"&gt;&lt;/span&gt;
-      &lt;span id="a1r2"&gt;No&lt;/span&gt;
-    &lt;/div&gt;
+      
+         <p>This example demonstrates use <code class="prop">role=radiogroup</code>.</p>
+         <div class="note">
+            <p>The radio buttons are custom controls that use <code class="prop">role=radio</code>, but the script to make them work like HTML radio buttons is not included in this example.</p>
```

Comment:
If we want to give the example of added script, we can add the [ARIA radio group pattern example](https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio-activedescendant.html) but it could be optional.